pandas/tests/groupby/test_groupby.py

```diff
@@ -249,11 +249,11 @@ def test_len():
 
 def test_basic_regression():
     # regression
-    T = [1.0 * x for x in lrange(1, 10) * 10][:1095]
-    result = Series(T, lrange(0, len(T)))
+    T = [1.0 * x for x in list(range(1, 10)) * 10][:1095]
+    result = Series(T)
```

Comment:
Small cleanup: Setting the index to ``lrange(0, len(T))`` is not needed as it's just the length of the series.